nixos/modules/services/misc/gitlab.nix

```diff
@@ -305,7 +296,6 @@ in {
 
       initialRootPassword = mkOption {
         type = types.str;
-        default = "UseNixOS!";
```

Comment:
I'd not remove this default from 18.09 as it might force people to update their config..

Reply:
Right, this would mean people would now then have to configure a (unused) `initialRootPassword` for deployments already active since a long time, which is silly. Updated.

Reply:
Thinking about this, what happens to the `db:seed_fu` task in nixos unstable, if the user doesn't explicitly configure one?

Reply:
Now or previously?
Now evaluation will not pass without specifying one, previously it would've used the default on the first run.

Reply:
Ok, I think that's fine for unstable.